libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/BotFrameworkHttpAdapter.cs

```diff
@@ -164,6 +164,7 @@ public async Task ProcessAsync(HttpRequest httpRequest, HttpResponse httpRespons
                 if (string.IsNullOrEmpty(activity?.Type))
                 {
                     httpResponse.StatusCode = (int)HttpStatusCode.BadRequest;
+                    Logger.LogInformation($"BadRequest: Activity is missing the Type field.");
```

Comment:
Can we do logWarning instead:
```suggestion
                    Logger.LogWarning("BadRequest: Missing activity or activity type.");
```